.pre-commit-config.yaml

```diff
@@ -985,6 +985,13 @@ repos:
         files: \.py$
         exclude: ^.*/.*_vendor/
         additional_dependencies: ['rich>=12.4.4']
+      - id: check-compat-cache-on-methods
+        name: Check that compat cache do not use on class methods
+        entry: ./scripts/ci/pre_commit/pre_commit_compat_cache_on_methods.py
```

Comment:
I have a PR to rename the pre-commit scripts so this might need to change later. Not a big problem though.

Reply:
You want to remove `pre_commit_` prefix? Because it is my main desire when I open this folder 😄

Reply:
Oh, yeah... I see https://github.com/apache/airflow/pull/38667 🥳

Reply:
Also remove that prefix from script file